### ISSUE TITLE
Does not work in Simulacrum

### ISSUE THREAD

**Issue by Rhimens** [mrchous/ExternalStartingItems#4]
Items set to be added to standard runs are not given to players at the start of Simulacrum runs.

**Comment by IKEA0x90** [comment]
Yeah, that's a know thing. The mod checks if the player drops in a drop pod when starting a run, which doesn't happen in Simulacrum. If I remove this check, nothing spawns on the stages for some reason. 
I'll leave this open in case I decide to look into it sometime, but no promises.
